OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/PrefManager.java

```diff
@@ -280,7 +285,7 @@ public void setIsVideosCacheRestored(boolean restored) {
     }
 
     /**
-     * Clears all the shared preferences being used in app.
+     * Clears all the shared preferences that are being used in app.
```

Comment:
[You are contradicting yourself.](https://github.com/edx/edx-app-android/pull/797#r76391499)